### PR TITLE
Add termcap/1.3.1 recipe

### PR DESCRIPTION
**Pull request by madebr**
Specify library name and version:  **termcap/1.3.1**

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/wiki#how-to-submit-a-pull-request) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Comment by madebr** [comment]
@uilianries @danimtb Can it have another go?

**Comment by uilianries** [comment]
It's Friday 13! Some JaSON is attacking Jenkins

**Comment by uilianries** [comment]
@madebr It's running again, let's wait for the result.

**Comment by uilianries** [comment]
@madebr take a look: https://github.com/madebr/conan-center-index/pull/4

File: recipes/termcap/all/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,39 @@
+cmake_minimum_required(VERSION 2.8.11)
```
Comment (danimtb) [inline recipes/termcap/all/CMakeLists.txt]: Why do you provide a custom cmakelist for this library?

File: recipes/termcap/all/conanfile.py [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+        cmake = self._configure_cmake()
+        cmake.build()
+
+    def package(self):
+        self.copy(pattern="COPYING", dst="licenses", src=self._source_subfolder)
+        cmake = self._configure_cmake()
+        cmake.install()
+
+    def package_info(self):
+        self.cpp_info.name = "Termcap"
```
Comment (danimtb) [inline recipes/termcap/all/conanfile.py]: Set the name to the specific generators
```suggestion
        self.cpp_info.names["cmake_find_package"] = "Termcap"
        self.cpp_info.names["cmake_find_package_multi"] = "Termcap"
```

File: recipes/termcap/all/patches/0001-msvc.patch [hunk 3]
```diff
@@ -0,0 +1,193 @@
+diff --git a/termcap.h b/termcap.h
```
Comment (danimtb) [inline recipes/termcap/all/patches/0001-msvc.patch]: This patch might be too much to make the library shared in MSVC. I am not sure if we should go that far

File: recipes/termcap/all/CMakeLists.txt (hunk 1 above)
Comment (madebr) [reply recipes/termcap/all/CMakeLists.txt]: To support Visual Studio

File: recipes/termcap/all/patches/0001-msvc.patch (hunk 3 above)
Comment (madebr) [reply recipes/termcap/all/patches/0001-msvc.patch]: The effort is done, working and amortized as of know.
So no problem imho.

File: recipes/termcap/all/CMakeLists.txt (hunk 1 above)
Comment (danimtb) [reply recipes/termcap/all/CMakeLists.txt]: Was this extracted from anywhere?

Comment (uilianries) [reply recipes/termcap/all/CMakeLists.txt]: good question, I made the original cmake file in Bincrafters: https://github.com/bincrafters/conan-termcap/blob/testing/1.3.1/CMakeLists.txt. However, this one I dunno where came from.

Comment (madebr) [reply recipes/termcap/all/CMakeLists.txt]: it's based only based on bincrafters' recipe.

File: recipes/termcap/all/patches/0001-msvc.patch (hunk 3 above)
Comment (SSE4) [reply recipes/termcap/all/patches/0001-msvc.patch]: let's keep it, as it's already done

**Pull request merged**